data/forcing/forcings-engine/.gitignore

```diff
@@ -0,0 +1,8 @@
+gfs/**/*.grib2
+
+# Temp directory used by the forcings engine instances
+scratch/*
+!scratch/.keep
+
+# Cropped GRIB files
+!gfs/gfs.20240117/00/atmos/gfs.t00z.sfluxgrbf*.grib2
```

Comment:
It would be nice if tests, examples, and such were configured in a way that these weren't going to be written in a path beneath the repository, but rather somewhere in the build tree. Is that a reasonable possibility?

Reply:
Yeah that makes sense. I think I can just configure the input config file via CMake -- should be fairly straightforward.